src/garage/torch/algos/ddpg.py

```diff
@@ -0,0 +1,300 @@
+"""This modules creates a DDPG model in PyTorch."""
+from collections import deque
+
+from dowel import logger, tabular
+import numpy as np
+import torch
+
+from garage.np.algos.off_policy_rl_algorithm import OffPolicyRLAlgorithm
+
+
+class DDPG(OffPolicyRLAlgorithm):
+    """A DDPG model implemented with PyTorch.
+
+    DDPG, also known as Deep Deterministic Policy Gradient, uses actor-critic
+    method to optimize the policy and reward prediction. It uses a supervised
+    method to update the critic network and policy gradient to update the actor
+    network. And there are exploration strategy, replay buffer and target
+    networks involved to stabilize the training process.
+
+    Args:
+        env_spec (EnvSpec): Environment specification.
+        policy (garage.torch.policies.base.Policy): Policy.
+        target_policy (garage.torch.policies.base.Policy): Copy of policy.
```

Comment:
i think that this copy should be maintained by the algorithm (derived from policy), rather than expecting the user to provide an exact copy of policy.